tasks/yesno.py

```diff
@@ -0,0 +1,301 @@
+
```

Comment:
Actually, it might make sense to have two models that are similar, but one uses the hypothesis aggregation like yesno while the other just averages pair predictions, for comparison purposes.  But they must at least be able to process the same dataset, so could we use the hypev format and load_hypev()?